packages/mobile/src/harmony-native/components/CompletionCheck/CompletionCheck.tsx

```diff
@@ -0,0 +1,94 @@
+import type { CompletionCheckProps } from '@audius/harmony'
+import styled, { css } from '@emotion/native'
+import type { ViewProps } from 'react-native'
+import Animated, {
+  Easing,
+  interpolate,
+  useAnimatedStyle,
+  useDerivedValue,
+  withTiming
+} from 'react-native-reanimated'
+
+import {
+  Flex,
+  IconValidationCheck,
```

Comment:
so nice we can reuse this. I think we should avoid @audius/harmony-native imports inside harmony-native though just headsup